tutorials/admin/getting-started/installation.md

```diff
@@ -1 +1,237 @@
+this before.
+
+(fqdn)=
+#### A fully qualified domain name (FQDN)
+
+You can usually purchase a domain name from one of the various domain name providers online. Search for "domain name
+provider" online. You may have to shop around a bit to find one that best fits your needs and budget.
+
+If you are hosting your Marble node in a cloud environment, most cloud service providers also provide a free or
+discounted domain name with your purchase of a cloud based virtual machine.
+
+(public-ip)=
+#### A static public IP address for your server
+
+A static public IP address is required so that the domain name server will know where to forward internet traffic to.
+
+If you are hosting your Marble node in a cloud based environment, follow the instructions from your cloud service 
+provider on how to forward internet traffic to your virtual machine.
+
+If you are installing Marble on premises at an institution, contact the institution's IT department to ask how to
+acquire your public static IP for the purposes of setting up a web application.
+
+```{note}
+Most residential internet service plans only provide *dynamic* public IP addresses, which are not suitable for hosting
+a web application. You may be able to request a static IP address from your internet service provider (ISP) for a small 
+fee depending on your ISP and/or geographical region.
+```
+
+(ssl-cert)=
+#### An SSL certificate for the FQDN
+
+An SSL certificate will encrypt all internet traffic to and from your server allowing Marble to use the HTTPS protocol.
+
+You may be able to acquire an SSL certificate from the same place that you purchased your [domain name](fqdn).
+
+Otherwise, you can acquire a certificate from organizations like [Let's Encrypt](https://letsencrypt.org).
+
+We recommend using a tool like [certbot](https://certbot.eff.org/) to get and manage SSL certificates if you are
+managing them yourself.
+
+```{note}
+If using [certbot](https://certbot.eff.org/), note that the Marble stack uses 
+[nginx](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/) as a reverse proxy.
+
+Certbot also requires that the website be already running and that ports 80 and 443 are open.
+To deploy the stack and start the website see the instructions [here](deploy-marble).
+```
+
+### Git
+
+The Marble source code is hosted on [GitHub](https://github.com) and a `git` client is highly recommended to download 
+and update the source code.
+
+Your server may already have `git` installed. To check run the following command from the command line:
+
+```shell
+git --version
+```
+
+The output should be some information about the version of `git` currently installed on your machine.
+
+If you do not have `git` installed, it can be installed from https://git-scm.com/downloads. Select your operating system
+for information on how to install `git`.
+
+## Marble
+
+Marble is deployed using a piece of software named [Birdhouse](https://github.com/bird-house/birdhouse-deploy/).
```

Comment:
I wouldn't call it a "piece of software", sounds pejorative :D 

How about "Marble is deployed using a node-management software named Birdhouse"?

Reply:
Also, I think we should add a nuance here about "Birdhouse" and "birdhouse-deploy".

While writing this, I noticed that the birdhouse project hasn't been consistent about using "birdhouse" vs "bird-house"...

Reply:
> I wouldn't call it a "piece of software", sounds pejorative :D

sure :shrug: 

> Also, I think we should add a nuance here about "Birdhouse" and "birdhouse-deploy".

What do you recommend? I'm not sure I know what you mean.

Reply:
This is probably pedantic, and a situation I feel we are in because of how Birdhouse organized itself. I was thinking something like: "Marble is built using [Birdhouse](link to birdhouse organization), which is a collection of ....., and is deployed using [birdhouse-deploy](....)" or something like that. I'm not married to it if you don't feel there is any need to change, I am actually good with that too.

Reply:
I like that wording, I'll change it up.